### PR TITLE
[Screenshots] added for both integration External Secrets Operator #5328 & Dex #5319

### LINKED ISSUE #5328: [Screenshots] External Secrets Operator under 'How it Works See It in Action' section
### Current Behavior
Currently, [every integration](https://layer5.io/cloud-native-management/meshery/integrations/external-secrets-operator) has only two old screenshots of the designer and visualizer in the 'How it Works See It in Action' section.

### Desired Behavior
- Add new images and videos from the latest version of the software ([MeshMap](https://layer5.io/cloud-native-management/meshmap))
- Add screenshots for [External Secrets Operator](https://layer5.io/cloud-native-management/meshery/integrations/external-secrets-operator) integration.

### Acceptance Tests
- Ensure your screenshot represents the use of a few specific integrations.
- The screenshot from the Designer mode should contain a valid design, refer to [meshery catalog](https://meshery.io/catalog) to use and view published designs.
- Add both Dark Mode and Light Mode screenshots.
- The screenshot should not be a snippet, take the full screenshot of your browser.
- Use Incognito mode on Chrome browser, to ensure your bookmark bar or any other items are not shown in the screenshot/video.

---

<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-  🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by meetjn**
**Description**

This PR fixes #5328 

**Notes for Reviewers**

Added both Dark Mode and Light Mode screenshots.

designer dark
designer light
visualizer dark
visualizer light

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits. 
- [x] Signed-off-by: Meet Jain <meetjaiin@gmail.com>

### CODE REVIEW

File: src/collections/integrations/_images/external secret/visualizer light.png
Comment (vishalvivekm) [inline src/collections/integrations/_images/external secret/visualizer light.png:1]: the folder should be named `external-secrets-operator`and not `external secret` 
will you please update it @meetjn

File: src/collections/integrations/external-secrets-operator/index.mdx [hunk 1]
```diff
@@ -33,8 +33,10 @@ featureList: [
   "Ongoing synchronization of Kubernetes configuration and changes across any number of clusters."
 ]
 workingSlides: [
-  ../_images/meshmap-visualizer.png,
-  ../_images/meshmap-designer.png
+  /Users/meet/Desktop/layer5/src/collections/integrations/_images/external secret/designer dark.png,
+  /Users/meet/Desktop/layer5/src/collections/integrations/_images/external secret/designer light.png,
+  /Users/meet/Desktop/layer5/src/collections/integrations/_images/external secret/visualizer dark.png,
+  /Users/meet/Desktop/layer5/src/collections/integrations/_images/external secret/visualizer light.png
```
Comment (vishalvivekm) [inline src/collections/integrations/external-secrets-operator/index.mdx:39]: These changes aren't needed @meetjn.
Revert these, if you would, please.

**Review by vishalvivekm — CHANGES_REQUESTED** [review]
preview is failing.
have suggested some changes, they might fix it.

**Comment by l5io** [comment]
🚀 Preview for commit 989750bd70252132271ff393d2805ff01cc48567 at: https://65d6ec7d0c8e5866d8c5dbcd--layer5.netlify.app

**Comment by meetjn** [comment]
> preview is failing. have suggested some changes, they might fix it.

Updated the suggested changes.

**Comment by meetjn** [comment]
> There're not changes @meetjn
> 
> image
> 
> Where're the screenshots and the directory I previously mentioned under `_images/`
> 
> https://65d6ec7d0c8e5866d8c5dbcd--layer5.netlify.app/cloud-native-management/meshery/integrations/external-secrets-operator

Updated screenshots with the file paths and new directory under _images

**Comment by l5io** [comment]
🚀 Preview for commit 821e386a72282b8ddb06795a471e1169a2808311 at: https://65d95c34145040971390d67c--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 9d0c238d902328b4c12e8ff4d343b0640a62748c at: https://65d96cb7fd55299b3dec9be6--layer5.netlify.app

File: .gitpod.yml
Comment (vishalvivekm) [inline .gitpod.yml:1]: please revert this change.

File: src/collections/integrations/_images/External-secrets-operator/designDark.png
Comment (vishalvivekm) [inline src/collections/integrations/_images/External-secrets-operator/designDark.png:1]: `external-secrets-operator` not `External-secrets-operator`

**Review by vishalvivekm — CHANGES_REQUESTED** [review]

**Pull request closed without merging**